ngraph/core/reference/include/ngraph/runtime/reference/convolution.hpp

```diff
@@ -324,3 +330,6 @@ namespace ngraph
         } // namespace reference
     }     // namespace runtime
 } // namespace ngraph
+
+// can't be removed currently due to arm-plugin dependency
+#include "ngraph/runtime/reference/convolution_backprop_data.hpp"
```

Comment:
Why cannot we change ARM plugin?

Reply:
it can be changed, but from my point of view it will be better handle it in separate ticket. Is this approach enough?